### DISCUSSION TITLE
Discretization of spectra in Orange

### DISCUSSION THREAD

**Discussion by Q-RS6** [Quasars/orange-spectroscopy#710] [Q&A] [answered]
Hello Orange community. I am currently using Orange to make PCA's on my infrared spectra. When I tried to input them (OPUS source file), I found out that the discretization pattern was not the same accross all my spectra (dataset 1 : discretization is 3998-3996-3994..., dataset 2 : 3997-3995-3993...). Is there a way to continuize the data (making it 3998-3997-3996...) or, alternatively, to correct the discretization in order to have the same values (both datasests : 3998-3996-3994...). Thanks a lot !

EDIT : By the way, I don't quite understand why my data is considered as discrete, since the spectra are recorded for all wavenumbers, from 4000cm-1 until 900cm-1...

EDIT : Problem solved ! Just used the interpolate widget (didn't quite know about the existence of this one lol), and all seem to work perfectly fine. Have a nice week-end !

**Comment by borondics** [comment, marked as answer]
Hi! Try to use the interpolate widget from the spectroscopy package.
